main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/logger"
+	"github.com/kevinpita/go-todo/database"
+	"github.com/kevinpita/go-todo/routes"
+	"github.com/narvikd/fiberparser"
+	"log"
+)
+
+func main() {
+	const addr = ":3000"
+
+	err := setupApp(addr)
+	if err != nil {
+		log.Fatalf("App could not be started on %v\n", addr)
```

Comment:
Through your app you're printing and returning errors with capital letters, please standardise those to minus.
I will mark them for you.